charts/cryostat/values.yaml

```diff
@@ -198,6 +198,16 @@ authentication:
     ## @param authentication.basicAuth.filename Key within Secret containing the properties file. The properties file should contain one user per line, with the syntax "user=passHex", where "user" is the username and "passHex" is the SHA-256 hash of the desired password **(Required if basicAuth is enabled)**
     filename: ""
 
+oauth2Proxy:
+  ## @param authentication.oauth2Proxy.enabled Whether Cryostat should use OAuth2 Proxy for authentication. When true, Cryostat will authenticate users via OAuth2 Proxy.
+  enabled: true
+  ## @param authentication.oauth2Proxy.htpasswdFile Path to the htpasswd file for additional user authentication
+  htpasswdFile: "templates/auth_proxy_htpasswd"
```

Comment:
I think this overlaps with the existing `.Values.authentication.basicAuth` configuration. We should keep that configuration, which allows the user to specify a secret that contains the basic auth credentials, and if they do that then we should mount that secret into the oauth2-proxy at this hardcoded location.